docs/DeveloperGuide.adoc

```diff
@@ -727,11 +727,69 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
+|`* * *` |user |clear my address book |refresh it quickly
+
+|`* * *` |user with many friends |list all friends with a certain tag |group my friend by tag easily
+
+|`* * *` |As a user |add a person with link:#blank-information[blank information] |add him even if I do not know some details (e.g his address)
+
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |get help information |know where is wrong whe I get errors
+
+|`* * *` |user |update contact's information |correct or add their information
+
+|`* * *` |user |list contacts |view who are my contacts
+
+|`* * *` |user |hide private details  |minimise the chance of someone else seeing them by accident
+
+|`* * *` |user |undo decision  |revert changes in case of mistake
+
+|`* * *` |lazy user |have auto complete when I type  |type less word
+
+|`* *` |user who share my device with others| password to lock my address book| prevent others looking at it
+
+|`* *` |frequent user |change the font and background colour |use address book comfortably
+
+|`* *` |frequent user |use non-case sensitive command |type command easily
+
+|`* *` | user with many friends |add multiple people at one time| save my time for adding
+
+|`* *` |user |sort contacts |view my list easily
+
+|`* *` |user |tag my contacts |remember who they are through tags
+
+|`* *` |careful user |log in with my id and password |keep my privacy
+
+|`* *` |elder user |adjust font size |see more clearly
+
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
+|`* *` |user |find a person by name |save time from browsing through the entire list
+
+|`* *` |forgettable user |stick some important people on the top|locate them quickly
+
+|`* *` |user |create events and assign contacts |keep track of who is involved in events
+
+|`* *` |user |export contact details |share with others or transfer to other device
+
+|`* *` |user |create filter using multiple tags |make a specific search using tags
+
+|`*` | frequent user |listen some background music |address book happily
+
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
+
+|`*` |user |add pictures to contacts |remember who they are through pictures
+
+|`*` |user |alter whether information is private  |make information open to everyone whenever I want
+
+|`*` |user |sort the list  |locate a person easily
+
+|`*` |user |clean AddressBook  |save time from deleting people one by one
+
+|`*` |user |drag person card |reorder the list easily
+
+|`*` |user |select font style |more comfortable to read
```

Comment:
You should follow the format for better understanding of the user story --> '**So that I can...** read more comfortably'.

This is present in several of your user stories as well, do review and improve on them!